### PR TITLE
Update podman guide

### PR DESCRIPTION
**Pull request by GaviniSumanth**
Fixed minor mistakes in the podman installation guide
Update the podman update guide.

### CODE REVIEW

File: content/docs/install/10. podman.md [hunk 1]
```diff
@@ -64,7 +58,8 @@ Still confused about Docker? Check out [this FAQ](/faq/server#im-still-confused-
 >  -v </path/to/config>:/config \
 >  -v </path/to/metadata>:/metadata \
 >  -v </path/to/audiobooks>:/audiobooks \
->  -v </path/to/podcasts>:/podcasts \
```
Comment (nichwall) [inline content/docs/install/10. podman.md:67]: ABS supports podcast libraries. We do not want to remove this from the example commands for setting up a server

Comment (GaviniSumanth) [reply content/docs/install/10. podman.md:67]: I have added the podcast volume mount in the podman guide.

**Comment by GaviniSumanth** [comment]
I changed the mount points in the container file to their corresponding lower case version to keep the mount points same as those in the docker guide.

**Comment by GaviniSumanth** [comment]
In the podman update guide, the podman pull command to pull the new image needs to be run as the user running the container. I am not sure if that part is clear in the guide or not. Should I mention it to make things clear or does the current update guide make sense?

**Comment by nichwall** [comment]
I'm not familiar with podman, but maybe it makes more sense to have two sections, one for rootless and one for rootful? Something like

Here's how you do it rootful
```
(rootful pull)
(rootful restart)
```
Altenatively, here's how you do it rootless
```
(rootless pull)
(rootless restart)
```

**Comment by GaviniSumanth** [comment]
Should I remove the manual update method to keep things simple?
or
Should I update the manual guide?

**Comment by nichwall** [comment]
I think it's fine to leave the manual instructions in. The instructions for some of the Linux install sections include more information as well.

Maybe you could say for a manual update to delete the container (using whatever commands), and then reinstall using the "install" section above? That way may confuse people more, though.

We're getting to the point in the docs where we should probably reorganize things into multiple pages for better flow, so if the information is already there it's easier to reorganize versus writing from scratch again later.

**Comment by GaviniSumanth** [comment]
Ok. I will update the manual guide to separate rootful and rootless sections.

Maybe removing the podman run section of the update guide will help shorten it a bit. I don't think anyone will update a podman  container that way because to run a podman container on boot you need to use a quadlet file or generate a systemd file (This method is deprecated now)

**Comment by GaviniSumanth** [comment]
I have split the podman update guide into rootful and rootless sections.

I have not removed the podman run update guide.

**Comment by GaviniSumanth** [comment]
Why is this pull request not merged? Are there any changes that I need to make? If there are any changes to be made, please let me know.

**Comment by nichwall** [comment]
Thanks for the reminder. There's a lot less traffic on this repo so don't see the PRs as often.

**Pull request merged**